lua/harpoon/config.lua

```diff
@@ -108,7 +108,7 @@ function M.get_default_config()
                     bufnr = vim.fn.bufnr(list_item.value, true)
                 end
                 if not vim.api.nvim_buf_is_loaded(bufnr) then
-                    vim.fn.bufload(bufnr)
+                    pcall(vim.fn.bufload, bufnr)
```

Comment:
we need to do something here if we cannot load the buffer

Reply:
When searching for this error, I found [a similar issue in the vim-project](https://github.com/vim/vim/issues/8333). Bram was suggesting to ignore the error, which made me feel confident enough to report it here :)
I didn't see any issues with this change and it only gave an error (for me) when encountering a swapfile-message/-error!?
I'm sure there's a cleaner way of archiving the same!? It probably involves doing a check before calling `bufload`!?
I'll have another look at it. Also, I'll try to add a test for it.